packages/components/src/components/data-list/data-list-row.jsx

```diff
@@ -4,8 +4,18 @@ import { NavLink } from 'react-router-dom';
 
 const DataListRow = props => {
     const [show_desc, setShowDesc] = React.useState(false);
-    const { destination_link, acion_desc, row_key, row_gap } = props;
-    const { rowRenderer, row, measure, is_scrolling, is_new_row } = props;
+    const {
+        acion_desc,
```

Comment:
Is that a typo?

Reply:
I don't know, I just sorted this variable alphabetically, didn't work on it.

Reply:
Can we correct it here?

Reply:
What's the correct version? 😆